### ISSUE TITLE
wildcard support on file copy is broken for the shell adaptor

### ISSUE THREAD

**Issue by andre-merzky** [radical-cybertools/saga-python#436]

**Comment by vivek-bala** [comment]
Is there a commit or PR where this is addressed? I couldn't find any PRs though.

**Comment by andre-merzky** [comment]
No, this needs implementation.  No driving use case exists - but it would also make some routines in RP simpler (profile fetching etc).

**Comment by vivek-bala** [comment]
Thanks for the info. I'm assuming needs implementation in the current version. I'm leaving this open for now.
